source/refined-github.ts

```diff
@@ -218,3 +218,4 @@ import './features/close-as-unplanned.js';
 import './features/locked-issue.js';
 import './features/visit-tag.js';
 import './features/prevent-comment-loss.js';
+import './features/no-pr-commit-notification.js';
```

Comment:
Name help welcome

Reply:
Correct. PR notifications now just point to the main conversation.

Reply:
Yeah... I can't think of anything that I like better. The best I could come up with is `link-pr-notification-to-main-thread` or `pr-notification-link-to-main-thread`, but they're rather long and unwieldy. Maybe `pr-notification-conversation-tab`? I'm not convinced any of these are an improvement over the current name.

Reply:
Yeah I've been steering away from long names because it makes mentioning them in commits painful.

`commit-range-not-found` would be nice because it explains why this feature exists at all.

I went with `pr-notification-link` because "it alters the link to PR notifications"